### DISCUSSION TITLE
feature selection

### DISCUSSION THREAD

**Discussion by imauzzc** [pycaret/pycaret#2918] [Q&A]
Hello, thank you very much for developing pycaret, which makes it easier for me to use python for data analysis. Recently I want to finish a paper using pycaret, which uses the feature selection function, and I want to understand how the function is implemented. I checked your website and github and found the following description: "It uses a combination of several supervised feature selection techniques to select the subset of features But I don't understand how it works, can you explain it a little? And I noticed this issue "Future Selection: Boruta algorithm #362", did Boruta finally get integrated into pycaret? I'm looking forward to hearing from you, thanks!

**Comment by tvdboom** [comment]
Hi. Boruta was part of pycaret until version 2.3.10. It is deprecated since version 3.0 because the underlying library was no longer actively maintained.

**Reply by imauzzc** [reply]
Thanks for your reply, so the current version 2.3.10, is the feature selection implemented by Boruta algorithm? Can you reveal which algorithm is used for feature selection in version 3.0?

**Reply by tvdboom** [reply]
https://github.com/pycaret/pycaret/blob/e22f10bb71f8b64ed577fafca87636d216285116/pycaret/classification/oop.py#L519-L541
